highlight/HighlightsFromPointsExplainer.md

```diff
@@ -0,0 +1,60 @@
+# Events for Highlighted Ranges
+
+## Status of this Document
+This document is intended as a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+* This document status: **Active**
+* Expected venue: [W3C CSS Working Group](https://www.w3.org/Style/CSS/)
+* Current version: this document
+
+## Introduction
+[Highlights](https://drafts.csswg.org/css-highlight-api-1/) give ranges an appearance, such as rendering background colors to denote comments or annotations in a document, find-on-page for virtualized content, and squiggles for spelling and grammar checking.
+
+These use cases require that the users not only be able to see the highlighted portion of the document, but have the ability to interact with it.
+
+Here are some inspirational examples of how users may interact with highlighted ranges:
+ - When a user clicks a highlighted result from find-on-page, the selection may be moved to cover the result so that it can be copied or edited easily.
+ - When a user hovers over a misspelled word, the web app may display UI with suggested replacement text.
+ - When a user clicks an annotation in a document, the web app may emphasize and scroll into view the corresponding annotation in a pane which lists all the annotations in the document.
+
+
+## Proposal
+The highlightsFromPoint() API is designed to enhance user interaction with highlighted content on web pages. Highlights are used to visually distinguish specific ranges within a document, such as comments, search results, or spelling errors. This API aims to provide a robust mechanism for identifying and interacting with these highlights, addressing challenges related to multiple overlapping highlights, event coordination, shadow DOM handling, and performance optimization.
+
+### Key Considerations 
+- **Multiple Overlapping Highlights**: When multiple highlights overlap (e.g., from a spell-checker and a find-on-page feature), it's crucial to identify all highlights at a specific point. This allows web developers to determine which highlight should receive the event first, ensuring the correct highlight responds to user interactions.
+- **Event Coordination**: Different features might create highlights independently. This API provides a way to coordinate events between these highlights, ensuring each highlight can handle events appropriately without conflicts.

```

Comment:
Not anymore since we're no longer considering a new event model. This point should be removed across the entire document.

Reply:
Removed all instances of the event model